### DISCUSSION TITLE
Docker Image Creation Fails

### DISCUSSION THREAD

**Discussion by botPotyo** [Flexget/Flexget#3459] [Q&A]
I'm trying to install Flexget using the Docker image method documented here: https://flexget.com/InstallWizard/SynologyNAS/Docker. 

However, the Docker image creation fails with the following error:
`Building wheels for collected packages: psutil, pynzb, pyrsistent, pyrss2gen, pyyaml, sgmllib3k, sqlalchemy, zxcvbn-python
  Building wheel for psutil (setup.py): started
  Building wheel for psutil (setup.py): finished with status 'error'
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [50 lines of output]
      /tmp/tmpbzokio0o.c:1:10: fatal error: linux/ethtool.h: No such file or directory
          1 | #include <linux/ethtool.h>
            |          ^~~~~~~~~~~~~~~~~
      compilation terminated.
`
I assume the base python:alpine image is missing some of the required dependencies. Anyone knows which version of th epython base image should I use? I tried a half a dozen and all either have this same error or some other error.

**Comment by aidan-** [comment]
The `Dockerfile` in that guide is missing the `linux-headers` dependency.  I have updated the docs to fix this and managed to build the docker container successfully.  Give that a go.

At the same time, you should see if any of [these more widely](https://flexget.com/InstallWizard/Docker) used images would be more suitable for your use case.
